dir.traversal.targets

```diff
@@ -140,7 +142,13 @@
 
       <ProjectsToTest>
        <AdditionalProperties Condition="'%(ProjectsToTest.TestTFMs)'!=''">TestTFMs=%(ProjectsToTest.TestTFMs);%(ProjectsToTest.AdditionalProperties)</AdditionalProperties>
-       <AdditionalProperties>TestTFM=$(FilterToTestTFM);%(ProjectsToTest.AdditionalProperties)</AdditionalProperties>
+      </ProjectsToTest>
+      <ProjectsToTest>
+        <!-- When TestTFMs is not set, and FilterToTestTFM is netcoreapp1.1 -->
+       <AdditionalProperties Condition="('%(ProjectsToTest.TestTFMs)'=='' Or $([System.String]::new('%(Project.TestTFMs)').Contains(';netcoreapp1.0;'))) And '$(FilterToTestTFM)'=='netcoreapp1.1'">TestTFM=netcoreapp1.0;%(ProjectsToTest.AdditionalProperties)</AdditionalProperties>
```

Comment:
This specific condition is for all of the projects today that don't specify a TestTFMs. For all of those, we were running against netcoreapp1.0 and with this new condition it will continue to do so. If TestTFMs is provided, then we use that as the TestTFM